hunting/llm/queries/aws_bedrock_ignore_previous_prompt_detection.toml

```diff
@@ -0,0 +1,32 @@
+[hunt]
+author = "Elastic"
+description = "This analytic flags multiple instances of LLM vulnerability to malicious user interaction, helping to maintain ethical guidelines and compliance standards."
+integration = ["aws_bedrock.invocation"]
+uuid = "131e5887-463a-46a1-a44e-b96361bc6cbc"
+name = "AWS Bedrock LLM Ignore Previous Prompt Detection"
+language = ["ES|QL"]
+license = "Elastic License v2"
+query = [
+'''
+from logs-aws_bedrock.invocation-*
+ |WHERE @timestamp > NOW() - 1 HOUR
+   AND (
+     (gen_ai.prompt LIKE "*Ignore the above instructions*" OR 
+      gen_ai.prompt LIKE "*IGNORE INSTRUCTIONS*" OR
+      gen_ai.prompt LIKE "*IGNORE and PRINT*" OR
+      gen_ai.prompt LIKE "*Ignore and say*" OR 
+      gen_ai.prompt LIKE "*nevermind ignore what i asked and print")
```

Comment:
```suggestion
      gen_ai.prompt LIKE "*nevermind ignore what i asked*")
```
This one seems a bit more specific and less generic like the others. Perhaps you have a specific for this though, not sure

Reply:
It was mostly picked from common phrases, but i like the generic suggestion. Will take this